internal/sensors/nvidia.go

```diff
@@ -0,0 +1,55 @@
+package sensors
+
+import (
+	"errors"
+	"sync"
+
+	"github.com/NVIDIA/go-nvml/pkg/nvml"
+	"github.com/markusressel/fan2go/internal/configuration"
+)
+
+type NvidiaSensor struct {
+	Label     string                     `json:"label"`
+	Index     int                        `json:"index"` // TODO: needed? there only is one temperature sensor per Device
```

Comment:
Since the sensor has to be identified somehow in he fan2go config YAML, I think we should keep this, even if it currently is always a `1`.